includes/Discovery.php

```diff
@@ -47,8 +47,19 @@ class Discovery {
 	public function init() {
 		add_action( 'web_stories_story_head', [ $this, 'print_metadata' ] );
 		add_action( 'web_stories_story_head', [ $this, 'print_schemaorg_metadata' ] );
-		add_action( 'web_stories_story_head', [ $this, 'print_open_graph_metadata' ] );
-		add_action( 'web_stories_story_head', [ $this, 'print_twitter_metadata' ] );
+
+		/**
+		 * Filters filter to enable / disable open graph data.
+		 *
+		 * @since 1.2.0
+		 * @param bool $enable_open_graph Enable / disable open graph. Default to true.
+		 */
+		$enable_open_graph = apply_filters( 'web_stories_enable_open_graph', true );
```

Comment:
After looking at this, I think we need more fine-grained filters for each of these methods.

Plugins like Yoast SEO need to disable all of them, while Jetpack only would need to disable Open Graph & Twitter output.

So there could be a few filters:

* `web_stories_print_metadata`
* `web_stories_print_schemaorg_metadata`
* `web_stories_print_open_graph_metadata`
* `web_stories_print_twitter_metadata`

Reply:
@swissspidy Is this what you had in mind - https://github.com/google/web-stories-wp/pull/5312/commits/2cc1e9e3296dc237632d487f71f67d360e681f1b ?

Reply:
Yep!

Reply:
Fixed.